packages/e2e-test-utils/src/click-block-toolbar-button.js

```diff
@@ -6,15 +6,25 @@ import { showBlockToolbar } from './show-block-toolbar';
 /**
  * Clicks a block toolbar button.
  *
- * @param {string} buttonAriaLabel The aria label of the button to click.
+ * @param {string} text The text string of the button label or content.
+ * @param {string} type The type of button - 'icon' or 'text'.
  */
-export async function clickBlockToolbarButton( buttonAriaLabel ) {
+export async function clickBlockToolbarButton( text, type = 'icon' ) {
```

Comment:
Technically, just because a button has an icon doesn't mean it uses an `aria-label`. I think it's possible that some buttons in the future may have both a visible label AND an icon. Perhaps we should change `icon` to `aria-label`?

And since we're technically checking for text in both cases, I think the `text` type should be renamed to `content`.

Reply:
Thanks, your suggested wording is clearer I think so switch to that.